go-kit/metrics/provider/librato/librato.go

```diff
@@ -416,7 +416,7 @@ func (h *Histogram) With(labelValues ...string) kmetrics.Histogram {
 
 func (h *Histogram) reset() {
 	// Not happy with this, but the existing histogram doesn't have a Reset.
-	h.h = gohistogram.NewHistogram(h.buckets)
+	h.h, _ = tdigest.New() // errors only happen if you pass in wrong options.
```

Comment:
Kill the _above_ comment (not happy...)

Make this comment: "`err` is only non-nil when passing options."